src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncIteratorMethodToStateMachineRewriter.cs

```diff
@@ -23,13 +23,16 @@ internal sealed class AsyncIteratorMethodToStateMachineRewriter : AsyncMethodToS
         private readonly AsyncIteratorInfo _asyncIteratorInfo;
 
         /// <summary>
+        /// Where should we jump to to continue the execution of disposal path.
```

Comment:
>Where [](start = 12, length = 5)

We usually implement a "stack" using the compiler's execution stack (i.e. "previous=current; current=newvalue; dostuff; current=previous;" and a simple variable rather than a stack data structure.
I have no problem using an explicit stack data structure - it is certainly more clear. But please say that it is a stack in the doc here.

Reply:
Now that you mention it, I think I'll change to use a simple variable (typical pattern) rather than an explicit stack data structure.